msticpy/data/queries/kql_mdatp_user.yaml

```diff
@@ -0,0 +1,80 @@
+metadata:
+  version: 1
+  description: MDATP Queries
+  data_environments: [MDATP]
+  data_families: [MDATP]
+  tags: ['user']
+defaults:
+  metadata:
+    data_source: 'user_events'
+  parameters:
+      start:
+        description: Query start time
+        type: datetime
+        default: -30
+      end:
+        description: Query end time
+        type: datetime
+        default: 0
+      add_query_items:
+        description: Additional query clauses
+        type: str
+        default: ''
+sources:
```

Comment:
Was think that you could make some of these more flexible by including default params for host_name, account_name, file_path and then doing some Kusto logic to exclude things that are not set by the user. I have an example in kql_sent_alert.list_related_alerts - it's a bit klunky but might save some typing in the future.